libbeat/api/npipe/listener_windows.go

```diff
@@ -65,6 +65,13 @@ func DialContext(npipe string) func(context.Context, string, string) (net.Conn,
 	}
 }
 
+// Dial create a Dial to be use with an http.Client to connect to a pipe.
+func Dial(npipe string) func(string, string) (net.Conn, error) {
+	return func(_, _ string) (net.Conn, error) {
+		return winio.DialPipe(npipe, nil)
+	}
+}
```

Comment:
Is this function actually being called from somewhere?

Reply:
I wanted to follow existing dialer and provide the required helper methods.